test/api/asset.spec.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-console */
+/* eslint-disable promise/always-return */
+import { Asset } from '../../src/lib/asset';
+import { stackInstance } from '../utils/stack-instance';
+import { TAsset } from './types';
+
+const stack = stackInstance();
+const assetUid = 'blt122ea52c5d4ddf19';
```

Comment:
Please remove assetUid from the asset.apec.ts file @nadeem-cs

Reply:
Resolved in [this](https://github.com/contentstack/contentstack-javascript/pull/110) PR.